app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -18,16 +18,14 @@ import org.feature.fox.coffee_counter.data.models.response.LoginResponse
 import org.feature.fox.coffee_counter.data.models.response.UserIdResponse
 import org.feature.fox.coffee_counter.data.models.response.UserResponse
 import org.feature.fox.coffee_counter.di.module.ApiModule
-import org.junit.After
-import org.junit.Before
-import org.junit.Rule
-import org.junit.Test
+import org.junit.*
```

Comment:
wildcard import

Reply:
Should be resolved in [7d3fdf2](https://github.com/Morphclue/code-camp-2/pull/69/commits/7d3fdf23d81ed0affcba616ba2463e220958c11c).